pkgs/development/python-modules/picos/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, numpy
+, cvxopt
+}:
+
+buildPythonPackage rec {
+  pname = "picos";
+  version = "1.2.0";
+
+  src = fetchFromGitLab {
+    owner = "picos-api";
+    repo = "picos";
+    rev = "b9c3105b31166c307296dee6e0c26cf6a0ab8723";
+    sha256 = "018xhc7cb2crkk27lhl63c7h77w5wa37fg41i7nqr4xclr43cs9z";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    cvxopt
+  ];
+
+  doCheck = true;
+  checkPhase = ''
+    python ${src}/test.py
```

Comment:
```suggestion
    ${python.interpreter} test.py
```

Reply:
Thanks for all reviews. I'm just curious, why do we need `${python.interpreter}`, and not simply `python`? In my mind, nix makes sure that the environment defined in checks contains the good python version, and libraries, and I'd be afraid that `${python.interpreter}` does not contain the newly compiled lib as it seems to be a generic variable. What happens exactly?

Reply:
`${python.interpreter}` is an alias for whatever python interpreter version is being used
`bin/${executable}`. So it's an absolute path to the interpreter which is preferred. Also the python environment is not always that pure.